include/swift/SymbolGraphGen/SymbolGraphOptions.h

```diff
@@ -35,6 +35,10 @@ struct SymbolGraphOptions {
   /// Emit members gotten through class inheritance or protocol default
   /// implementations with compound, "SYNTHESIZED" USRs.
   bool EmitSynthesizedMembers;
+  
+  /// Whether to silence the "Found N symbols" messages when rendering
+  /// a symbol graph.
+  bool QuietMessages;
```

Comment:
Is "quiet" common terminology? Maybe `SilenceMessage` would fit better here (see https://github.com/apple/swift/blob/a5a79f23c0257c36c297d3c1b9275a873434468a/lib/Sema/TypeCheckType.h#L57)

Reply:
When i started writing this, i assumed i would use a `--quiet` flag, which i've seen on other tools to reduce/eliminate the messages they display. Since the meaning was ultimately inverted (no output without `-v` flag), this might work better as `PrintMessages` instead. What do you think?

Reply:
`PrintMessages` sounds good as well!